sequencing/test/test_system.py

```diff
@@ -75,6 +75,30 @@ def test_set_cross_kerr(self):
         self.assertEqual(len(system.couplings()), 1)
         self.assertEqual(system.couplings()[0], 2 * np.pi * chi * qubit.n * cavity.n)
 
+    def test_set_sixth_order_cross_kerr(self):
```

Comment:
This should be removed; updates are in this PR: https://github.com/qci-chen/sequencing/pull/1